src/radical/pilot/task_overlay/master.py

```diff
@@ -227,23 +229,31 @@ def wait(self, count=None, uids=None):
         if count:
             self._log.debug('wait for %d workers', count)
             while True:
+                stats = {
```

Comment:
just a stylistic comment - should we call it `statuses`? `stats` is referred to as `statistics`

Reply:
Ha, the `stats` is about gathering crude statistics: how many workers found in what status :-)

Reply:
ah... right!